sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -317,7 +317,7 @@ async Task<IEnumerable<T>> IAsyncConverter<MultiBlobContext, IEnumerable<T>>.Con
                 // sub directories.
                 string prefix = context.Prefix;
                 var container = context.Container;
-                IAsyncEnumerable<BlobItem> blobItems = container.GetBlobsAsync(prefix: prefix, cancellationToken: cancellationToken);
+                IAsyncEnumerable<BlobItem> blobItems = container.GetBlobsAsync(BlobTraits.None, BlobStates.None, prefix, cancellationToken);
```

Comment:
@seanmcc-msft @amnguye the breaking changes to the Blobs library required updating the WebJobs extension package as the project ref runs were failing otherwise.